cli/sql.go

```diff
@@ -271,6 +272,15 @@ func runTerm(cmd *cobra.Command, args []string) error {
 	if len(cliContext.execStmts) > 0 {
 		// Single-line sql; run as simple as possible, without noise on stdout.
 		return runStatements(conn, cliContext.execStmts)
+	} else if !isatty.IsTerminal(os.Stdin.Fd()) {
```

Comment:
have you encountered any issues with our line parsing? (eg: looking for a `;\n` to terminate a statement)
I can do the same thing right now, and leave proper parsing as a todo.